test_runner/src/main/kotlin/ftl/args/IArgs.kt

```diff
@@ -51,6 +51,12 @@ interface IArgs {
 
     fun useLocalResultDir() = localResultDir != FlankYmlParams.defaultLocalResultsDir
 
+    fun fixMaxTestShardsValue(inputValue: Int): Int = if (inputValue == -1) {
```

Comment:
I am not convinced with method name, it indicates some problems and does not tell what is wrong. First that came up to my mind is `checkIfInRangeOrDefault` but I am sure there is a better name

Reply:
Or maybe `convertToShardCount`?

Reply:
Thanks, changed 😄